src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -0,0 +1,115 @@
+using PKSim.CLI.Core.MinimalImplementations;
+using PKSim.Core;
+using PKSim.Core.Services;
+using PKSim.Infrastructure;
+using PKSim.Presentation;
+using PKSim.Presentation.DTO.Mappers;
+using PKSim.Presentation.Presenters.ExpressionProfiles;
+using PKSim.Presentation.Services;
+using PKSim.Presentation.Views.ExpressionProfiles;
+using PKSim.UI.Views.ExpressionProfiles;
+
+namespace PKSim.UI.Starter
+{
+   public static class ApplicationStartup
+   {
+      static IContainer _container;
+
+      public static IContainer Initialize(IShell shell)
+      {
+         if (_container != null)
+            return _container;
+
+         _container = InitializeForStartup(shell);
+
+         return _container;
+      }
+
+      public static IContainer InitializeForStartup(IShell shell)
+      {
+         Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+         Thread.CurrentThread.CurrentUICulture = new CultureInfo("en");
+         ApplicationIcons.DefaultIcon = ApplicationIcons.PKSim;
+
+
+         var container = InfrastructureRegister.Initialize(registerContainerAsStatic: false);
+         container.RegisterImplementationOf(SynchronizationContext.Current);
+
+         container.Register<IApplicationController, ApplicationController>(LifeStyle.Singleton);
+         container.Register<ICoreWorkspace, OSPSuite.Core.IWorkspace, IWorkspace, Workspace>(LifeStyle.Singleton);
+
+         // registerImplementationsFromCore(container);
+         registerMinimalImplementations(container);
+
+
+         using (container.OptimizeDependencyResolution())
+         {
+            container.RegisterImplementationOf(shell);
+            container.AddRegister(x => x.FromType<CoreRegister>());
+            container.AddRegister(x => x.FromType<InfrastructureRegister>());
+            container.Register<ICreateExpressionProfilePresenter, CreateExpressionProfilePresenter>();
+            container.Register<ICreateExpressionProfileView, CreateExpressionProfileView>();
+            container.Register<IExpressionProfileToExpressionProfileDTOMapper, ExpressionProfileToExpressionProfileDTOMapper>();
+            container.Register<IMoleculePropertiesMapper, MoleculePropertiesMapper>();
+            container.Register<ICoreUserSettings, IPresentationUserSettings, UIStarterUserSettings>(LifeStyle.Singleton);
+            container.Register<IOntogenyTask, OntogenyTask>();
+            container.Register<IEntityTask, EntityTask>();
+            container.Register<IRenameObjectDTOFactory, RenameObjectDTOFactory>();
+            container.Register<IHistoryManager, HistoryManager<IExecutionContext>>();
+            InfrastructureRegister.LoadSerializers(container);
+         }
+
+         return container;
+      }
+
+      // /// <summary>
+      // /// Use this method to register all dependencies in Core.UI and Core.Presentation
+      // /// </summary>
+      // private static void registerImplementationsFromCore(IContainer container)
```

Comment:
I left this in because I know I'll be working on this more for Individuals. Once we have the feature working for both, I can remove this full registration code instead of commenting it out.